tests/unit/lms/services/h_api_test.py

```diff
@@ -22,6 +24,40 @@ class TestHAPI:
     # protected-access messages.
     # pylint: disable=protected-access
 
+    def test_bulk_action_process_commands_correctly(self, h_api, BulkAPI):
+        h_api.bulk_action(
+            [sentinel.command_1, sentinel.command_2,]
+        )
+
+        BulkAPI.to_string.assert_called_once_with(
+            [
+                Any.object(ConfigCommand).with_attrs(
+                    {
+                        "body": Any.object.with_attrs(
+                            {
+                                "effective_user": "acct:lms@TEST_AUTHORITY",
+                                "total_instructions": 3,
+                            }
+                        )
+                    }
+                ),
+                sentinel.command_1,
+                sentinel.command_2,
+            ]
+        )
+
+    def test_bulk_action_calls_h_correctly(self, h_api, BulkAPI, _api_request):
+        h_api.bulk_action([sentinel.command])
+
+        _api_request.assert_called_once_with(
+            "POST",
+            path="bulk",
+            body=BulkAPI.to_string.return_value,
+            headers=Any.mapping.containing(
+                {"Content-Type": "application/vnd.hypothesis.v1+x-ndjson"}
+            ),
+        )
```

Comment:
Not much to say except it passes it on to BulkAPI correctly. Our testing there should get us the rest of the way.